### PR TITLE
Various updates to pipelines and pyproject

### PR DESCRIPTION
**Pull request by andped10**
Changed to `EasyScience` in `pyproject.yml`

Update github action for: `ossar-action@v2.0.0`

Upgrade pip for python-ci

Renaming pipelines:
- `release-drafter.yml`
- `release-drafter-verify-pr-labels.yml`

### CODE REVIEW

File: template/pyproject.toml.jinja [hunk 1]
```diff
@@ -28,7 +28,7 @@ classifiers = [
 ]
 requires-python = ">={{min_python}},<{{max_python}}"
 dependencies = [
-    "easyScienceCore>=0.3.1",
+    "EasyScience>=0.6.0",
```
Comment (rozyczko) [inline template/pyproject.toml.jinja]: or we can wait a bit to get that to be `easyscience` ...

**Review by rozyczko — COMMENTED** [review]
looks simple but we might need to wait a bit for easyscience to catch up namewise...

File: template/pyproject.toml.jinja (hunk 1 above)
Comment (andped10) [reply template/pyproject.toml.jinja]: changed to `easyscience`

**Review by rozyczko — APPROVED** [review]
this should now work as expected, with `easyscience` (lower case) being on pypi.

**Pull request merged**